inference-engine/src/transformations/src/transformations/common_optimizations/optimize_strided_slice.cpp

```diff
@@ -225,6 +225,11 @@ bool ngraph::pass::GroupedStridedSliceOptimizer::run_on_function(std::shared_ptr
         NodeVector ops_to_replace;
         for (auto & record : output_to_size) {
             if (record.first != fake_output) {
+                const auto out_name = record.first.get_node_shared_ptr()->get_friendly_name();
+                NGRAPH_SUPPRESS_DEPRECATED_START
+                variadic_split->get_output_tensor(i).set_name(out_name);
```

Comment:
It is a WA.
Can we use the [new API](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/include/cpp/ie_cnn_network.h#L199)?

It should allow to map framework names to OV names without work arounds on the transformations side.